core-bundle/src/Routing/ResponseContext/ResponseContext.php

```diff
@@ -52,6 +73,11 @@ public function has(string $serviceId): bool
         return isset($this->current[$serviceId]);
     }
 
+    public function isInitialized(string $serviceId): bool
```

Comment:
I've introduced that to improve performance. When setting data, this is not relevant (`has()` is enough). But if you want to call methods on a service like we do when we finally render the JSON-LD, we need this. This is to cover the use case where the controller has provided the `JsonLdManager` with `addLazy()` but no module/content element or whatever ever accessed it (= it was never initialized).
We would then intantiate the service only to see that it was empty. With the additional `isInitialized()` we can improve this :)